QUBEKit/dihedrals.py

```diff
@@ -77,56 +86,99 @@ def find_scan_order(self):
         if self.molecule.scan_order:
             return self.molecule
 
-        elif self.molecule.rotatable is None:
-            print('No rotatable torsions found in the molecule')
-            self.molecule.scan_order = []
-
-        elif len(self.molecule.rotatable) == 1:
-            print('One rotatable torsion found')
-            self.molecule.scan_order = self.molecule.rotatable
-
-        else:
-            # Get the rotatable dihedrals from the molecule
-            rotatable = list(self.molecule.rotatable)
-            print('Please select the central bonds round which you wish to scan in the order to be scanned')
-            print('Torsion number   Central-Bond   Representative Dihedral')
-            for i, bond in enumerate(rotatable):
-                print(f'  {i + 1}                    {bond[0]}-{bond[1]}             '
-                      f'{self.molecule.atoms[self.molecule.dihedrals[bond][0][0]].atom_name}-'
-                      f'{self.molecule.atoms[self.molecule.dihedrals[bond][0][1]].atom_name}-'
-                      f'{self.molecule.atoms[self.molecule.dihedrals[bond][0][2]].atom_name}-'
-                      f'{self.molecule.atoms[self.molecule.dihedrals[bond][0][3]].atom_name}')
-
-            scans = input('>').split()
-
-            scan_order = []
-            # Add the rotatable dihedral keys to an array
-            for scan in scans:
-                scan_order.append(rotatable[int(scan) - 1])
-            self.molecule.scan_order = scan_order
+        # Get the rotatable dihedrals from the molecule
```

Comment:
I think this logic should also be done in ligand we are deciding which bonds are rotatable, non-rotatable and improper here they should already be defined before we get here.